js/scoreFormatter.js

```diff
@@ -70,9 +72,26 @@ YoastSEO.ScoreFormatter.prototype.outputScore = function() {
  * sorts the scores array on ascending scores
  */
 YoastSEO.ScoreFormatter.prototype.sortScores = function() {
+	var unsortables = this.getUndefinedScores( this.scores );
```

Comment:
Doesn't this keep the undefined scores in the array duplicating all undefined scores?